src/terminal/Terminal.h

```diff
@@ -649,6 +657,8 @@ class Terminal
         [[nodiscard]] int cellWidth(CellLocation _pos) const noexcept override;
     };
     SelectionHelper selectionHelper_;
+    std::optional<CellLocationRange> highlightRange_ = std::nullopt;
+    std::chrono::milliseconds highlightTimeout_;
```

Comment:
It works with life config, but changes only get picked up on first save of config. On later saves no changes get picked up. Maybe it's a bug in live config because same thing happen with other options as well.

Reply:
Okay, I try to keep live reload working, but I'm aware that it's either not possible for all the config flags or it's not applied. No worries.